LunrCore/LunrCore.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;net5.0</TargetFrameworks>
     <RootNamespace>Lunr</RootNamespace>
     <Version>2.3.8.5</Version>
```

Comment:
Should we bump the version number here?

Reply:
Bumped to match lunr.js. It looks like your set union fix made that release.